packages/daf-w3c/src/action-handler.ts

```diff
@@ -109,54 +114,80 @@ export interface PresentationInput {
   tag?: string
   expirationDate?: string
   verifiableCredential: string[]
+  [x: string]: any
 }
 
 const transformCredentialInput = (input: CredentialInput): VerifiableCredentialPayload => {
```

Comment:
Are we dropping custom claims outside of the `credentialSubject`, e.g., 

```
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://www.w3.org/2018/credentials/examples/v1"
  ],
  "id": "http://example.edu/credentials/1872",
  "type": ["VerifiableCredential", "AlumniCredential"],
  "issuer": "https://example.edu/issuers/565049",
  "issuanceDate": "2010-01-01T19:73:24Z",
  "customProperty": "12345",
  "credentialSubject": {
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    "alumniOf": {
      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1",
    }
  }
```

Reply:
we were before, but no longer after this PR